lib/core/device.rb

```diff
@@ -461,6 +461,27 @@ def click(action)
     end
   end
 
+  # looks for a native iOS alert (within the springboard application). If found,
+  # clicks the specified alert button. Does nothing if the alert is not found.
+  # Accepts:
+  #   Strategy
+  #   Id
+  #   Condition
+  #   CheckTime
+  #   AlertTime
+  #   NoRaise
+  def handle_ios_alert(action)
+    alert_time = action.key?("AlertTime") ? action["AlertTime"] : 1
+    @driver.update_settings({ defaultActiveApplication: "com.apple.springboard" })
```

Comment:
I just wonder, shouldnt we just create a method for `update_settings` ? because this feels veeeery specific action, and not sure if we should generalise it and then the user will do something like:
```
- Type: update_settings
Value: { defaultActiveApplication: "com.apple.springboard" }
- Type: click
...
- Type: update_settings
Value: { defaultActiveApplication: "auto" }
```

What do you think?

Reply:
While I agree that update_settings would also be highly useful to have (and should be added), this particular case would be greatly simplified by merging 3 actions in one.
Arguably with xcuitest driver 7.17 this active app switching may not be necessary if using the `respectSystemAlerts` setting.

Reply:
> While I agree that update_settings would also be highly useful to have (and should be added), this particular case would be greatly simplified by merging 3 actions in one. Arguably with xcuitest driver 7.17 this active app switching may not be necessary if using the `respectSystemAlerts` setting.

then, can you create an `Enhancement ticket` adding this new type?

Reply:
Created #56